src/main/java/com/avengers/gamera/controller/ArticleController.java

```diff
@@ -26,4 +28,31 @@ public class ArticleController {
     public ArticleGetDto createArticle(@RequestBody ArticlePostDto articlePostDto) {
         return articleService.createArticle(articlePostDto);
     }
+
+    @PatchMapping("/{articleId}")
+    @Operation(summary = "Update article by article id")
+    @ResponseStatus(HttpStatus.OK)
+    public ArticleGetDto updateArticle(@RequestBody ArticlePatchDto articlePatchDto, @PathVariable Long articleId){
```

Comment:
Why do we need to individually set articleId as path variable rather than directly put it in the request body?